db/schema.rb

```diff
@@ -50,6 +50,7 @@
     t.string "prisoner_category"
     t.string "sentence_length"
     t.jsonb "complex_cases"
+    t.integer "moves_count", null: false
```

Comment:
Unrelated to PR

Reply:
Weirdly moved, yeah.

Reply:
They do that all the time - I try to avoid adding those lines to the index but it is an annoying aspect of migrations.. seems over time everyone ends up with a slightly divergent column order in postgres.